crypto/x509/x509_vfy.c

```diff
@@ -3268,6 +3268,49 @@ static int build_chain(X509_STORE_CTX *ctx)
     return 0;
 }
 
+STACK_OF(X509) *OSSL_build_cert_chain(OSSL_LIB_CTX *libctx, const char *propq,
```

Comment:
I think this fits in the X509 namespace.  And I don't know the conventions, but shouldn't `libctx` and `propq` be at the end?

And if you put `store` first, that argues that this is an `X509_STORE` function.

Reply:
And judging by the way it works `cert` should be before `inter`

Reply:
Good points!
So I've improved the function name and argument list.